zellij-server/src/tab/mod.rs

```diff
@@ -1409,6 +1409,22 @@ impl Tab {
             }
         })
     }
+    pub fn get_active_pane_details(&mut self, client_id: ClientId) -> Option<Vec<u8>> {
+        let mut bytes = Vec::new();
+        if let Some(pane) = self.get_active_pane(client_id) {
+            if let PaneId::Terminal(pid) = pane.pid() {
+                let msg = format!(
+                    "client #{} : Current active pane id is {}. <WIP>",
+                    client_id, pid
+                );
+                bytes.append(&mut msg.as_bytes().to_vec());
+
+                return Some(bytes);
+            }
+        }
+
+        None
+    }
```

Comment:
Can we add an option to output json so its parsable?

Reply:
Sorry for the last update, my team has been working hard for our new product release, so did not get a lot of time :)

I've added an optional argument to support json format

<img width="545" alt="image" src="https://github.com/zellij-org/zellij/assets/85712372/1cfc9018-5f99-4751-ad28-e98d31a073f9">

Reply:
No worries and thank you!